### ISSUE TITLE
Interesting thing happened

### ISSUE THREAD

**Issue by AymenKcm1** [sourcelocation/TrollTools#12]
Hidden system apps showed up after applying a theme. 

https://streamja.com/leB9e

Most of them work but others open to a black screen or crash.

Interesting Screenshots app & options in it : Multiple scrolling screenshots 🤔

![image](https://user-images.githubusercontent.com/67245917/199132030-31e1e475-c35f-4cc5-8d8f-af09283bf4a8.png)

How did this happen?

**Comment by sourcelocation** [comment]
interesting...

**Comment by corruptedkiwi** [comment]
Yeah I had what I assume is a similar thing, where the theme added a weather app to my iPad, which doesn’t have a native weather app on iOS 15. The themed app didn’t open

**Comment by zxsleebu** [comment]
i got xcode preview after applying a theme.
also thank you so much for contributing into open source.
it's an incredible work for a 15 yo.
i'm 16, also a private developer, but you have got a lot bigger potential than me!
